frontend/package.json

```diff
@@ -3,9 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material": "^5.14.8",
```

Comment:
No, it won't work, we won't use 3 libraries for pagination, and a big one like mui